software/builder/visionBuilder

```diff
@@ -134,36 +134,55 @@ function doLinkSrc() {
 
 # Get make options.
 function getMakeOpts() {
-  [[ "$CPUCOUNT" != "" ]] && return;
-  
+  MAKEOPTS=""
+
+  # Set our expectations regarding 'make' on this platform...
   local OS=`uname`
 
-  # Find our CPU count, default to 1.
-  CPUCOUNT=1
   case $OS in
   Linux)
+    [[ ! -v CPUCOUNT ]] && CPUCOUNT=$(grep -c processor /proc/cpuinfo)
```

Comment:
No `-v` in `bash` prior to 4.2.  I'm trying [this workaround](https://unix.stackexchange.com/questions/56837/how-to-test-if-a-variable-is-defined-at-all-in-bash-prior-to-version-4-2-with-th)